pkgs/data/themes/whitesur/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, glib
+, gnome-themes-extra
+, libxml2
+, sassc
+, sudo
+, util-linux
+, which
+}:
+
+stdenv.mkDerivation rec {
+  pname = "whitesur-gtk-theme";
+  version = "2021-06-23";
+
+  src = fetchFromGitHub {
+    owner = "vinceliuice";
+    repo = pname;
+    rev = version;
+    sha256 = "075fw57mv6zadq4dryn8bg2b3vq8inmisq18s758cv53pprxh9hw";
+  };
+
+  nativeBuildInputs = [
+    glib
+    libxml2
+    sassc
+    sudo
```

Comment:
Why sudo?

Reply:
The installation script fails if `sudo` is not found.

https://github.com/vinceliuice/WhiteSur-gtk-theme/blob/master/lib-core.sh#L46

Although it is not used in our use case of the install script, I think.

Reply:
I would prefer your suggestion instead of providing sudo here.

Reply:
Done.